Apps/Admin/Server/appsettings.json

```diff
@@ -139,5 +139,13 @@
     "Delegation": {
         "MaxDependentAge": 12,
         "MinDelegateAge": 12
+    },
```

Comment:
I think this is redundant, if the settings default to false and not specified, it would still be false

Reply:
Can discuss again with @sslaws